packages/services/src/workers/notification-scanner.js

```diff
@@ -0,0 +1,33 @@
+import { User, UserNotification, UserNotificationType } from '@aragonone/court-backend-server/build/models/objection'
+import * as notificationScanners from '../models/notification-scanners'
+
+/**
+ * This worker loops over all notification scanner objects and 
+ * inserts a notification DB entry for every email that should be sent
+ */
+export default async function (logger) {
+  Object.keys(notificationScanners).forEach(async (model) => {
+    await tryRunScanner(logger, model)
+  })
+}
+
+export async function tryRunScanner(logger, model) {
+  const type = await UserNotificationType.findOrInsert({model})
+  const scanner = notificationScanners[model]
```

Comment:
we can add a validation here since this is probably going to fail if someone renames a file or adds a new type in the DB manually